hibernate-core/src/test/java/org/hibernate/test/type/descriptor/java/LocaleTypeDescriptorTest.java

```diff
@@ -47,6 +48,9 @@ public Locale toLocale(String lang, String region, String variant) {
 		if ( StringHelper.isNotEmpty( variant ) ) {
 			builder.setVariant( variant );
 		}
+		if ( StringHelper.isNotEmpty(script) ) {
+			builder.setScript(script);
```

Comment:
it would be great to add space chars around `script` to adhere to existing coding format.